pkgs/top-level/python-aliases.nix

```diff
@@ -1,4 +1,4 @@
-lib: self: super:
+{ lib, fetchPypi }: self: super:
```

Comment:
```suggestion
{ lib, pkgs }: self: super:
```
if we add any other packages here in the future

Reply:
Honestly I am not sure and if it breaks splicing. I don't know if we have a predecessor for this. Maybe it is enough to use a throw like the other packages we move to pkgs.

Reply:
The package set should already be part of `super`, so `super.pkgs.fetchPypi`.

Reply:
good point, updated the PR